common/tools/dev-tool/test/customization/exports.spec.ts

```diff
@@ -38,14 +41,20 @@ describe("Exports", () => {
 
     augmentExports(customFile, originalFile);
 
-    expect(originalFile.getExportDeclarations()).to.have.lengthOf(1);
+    assert.equal(originalFile.getExportDeclarations().length, 1);
 
     const exportDeclaration = originalFile.getExportDeclarations()[0];
-    expect(exportDeclaration.getModuleSpecifier()?.getLiteralValue()).to.equal("./module");
+    assert.equal(exportDeclaration.getModuleSpecifier()?.getLiteralValue(), "./module");
 
     const namedExports = exportDeclaration.getNamedExports();
-    expect(namedExports).to.have.lengthOf(2);
-    expect(namedExports.map((e) => e.getName())).contains("Foo");
-    expect(namedExports.map((e) => e.getName())).contains("Bar");
+    assert.equal(namedExports.length, 2);
+    assert.include(
```

Comment:
nit: `assert.includeMembers` if you wanted to combine these and avoid the double map operation ;)

Reply:
Fixed